ci/code_checks.sh

```diff
@@ -184,6 +184,11 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     invgrep -R --include="*.rst" ".. ipython ::" doc/source
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    # Check for blank lines after the class definition
+    MSG='Check for extra blank lines after the class definition' ; echo $MSG
+    invgrep -R --include="*.py" --include="*.pyx" -E 'class.*:\n\n( )+"""' doc/source/
```

Comment:
```suggestion
    invgrep -R --include="*.py" --include="*.pyx" -E 'class.*:\n\n( )+"""' .
```

The goal here is to check all the Python files in the repo, to standardize classes. The last path `doc/source/` is checking only in the documentation directory, where I don't think there are python files, so we want to look in the root instead